pkg/certificate/errors.go

```diff
@@ -7,5 +7,7 @@ import (
 var errEncodeKey = errors.New("encode key")
 var errEncodeCert = errors.New("encode cert")
 var errMarshalPrivateKey = errors.New("marshal private key")
-var errNoCertificateInPEM = errors.New("no certificate in PEM")
 var errNoPrivateKeyInPEM = errors.New("no private Key in PEM")
+
+// ErrNoCertificateInPEM is the errror for no certificate in PEM
+var ErrNoCertificateInPEM = errors.New("no certificate in PEM")
```

Comment:
Is there a reason this is public?

Reply:
I think because it is used in another package, `certmanager`, for testing purpose.